doc/changes/latest.rst

```diff
@@ -12,6 +12,7 @@ NEW
 
 - ``fsaverage`` meshes accessed through :func:`~datasets.fetch_surf_fsaverage` now come with flat maps for all resolutions (:gh:`3444` by `Alexis Thual`_).
 
+- surface plotting functions allow setting custom view angles and are no longer limited to predefined views (:gh:`3259` by `Sam Buck Johnson`_ and `Alexis Thual`_).
```

Comment:
You'll need to also add @sambjohnson to `doc/changes/names.rst` because of being a first-time contributor or else this link will error out